features/providers/bank_statement_flow.feature

```diff
@@ -40,6 +40,21 @@ Feature: Bank statement flow
     When I select 'Housing payments'
     But I select 'None of the above'
     And I click 'Save and continue'
+    Then I should be on a page showing "Sort your client's income into categories"
+
+    Then I should be on the 'income_summary' page showing "Sort your client's income into categories"
+    And I click the first link 'View statements and add transactions'
+    Then I should be on a page showing 'Select benefits payments'
+    Then I select the first checkbox
+    And I click 'Save and continue'
+    And I click 'Save and continue'
+    Then I should be on the 'outgoings_summary' page showing "Sort your client's regular payments into categories"
+
+    When I click the first link 'View statements and add transactions'
+    Then I should be on a page showing 'Select housing payments'
+    When I select the first checkbox
+    And I click 'Save and continue'
+    And I click 'Save and continue'
```

Comment:
This is not the correct flow. bank statements upload journey should not be sorting transactions supplied by the truelayer journey, no? I noticed this when UATing first

Reply:
I have fixed this in 3366